x-pack/plugins/ml/common/util/job_utils.ts

```diff
@@ -916,3 +916,20 @@ export function isKnownEmptyQuery(query: QueryDslQueryContainer) {
 
   return false;
 }
+
+/**
+ * Extract unique influencers from the job or collection of jobs
+ * @param jobs
+ */
+export function extractInfluencers(jobs: Job | Job[]): string[] {
```

Comment:
moved from `AnomalyDetectorService` as it doesn't need to live inside the class.